content/blog/sivayanama-simplifying-open-api-specification-with-reusable-common-data-models.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Simplifying open API specification with reusable common data models
+date: 2023-08-26T08:06:19.141Z
+author: BalaSubramanian Vetrivel
+authorimage: /img/vetrivel-balasubramanian1-photo.jpg
+disable: false
+---
+The open API specification is one of the most widely followed API contracts. It is language-agonistic.
+With the help of these Open API specifications, clients can understand the API and invoke them without having access to the code or worrying about the implementation details. At times, this open specification file is becoming too complex to manage and understand. In this article, I will discuss the techniques to simplify the open specification with loosely coupled reusable data models.
```

Comment:
You should separate before "At times", that this is for API authors, because the previous sentence is about API consumers.